contributing.md

```diff
@@ -18,10 +20,17 @@ Example for .NET:
 
 ```
 $ git clone git@github.com:username/azure-activedirectory-library-for-dotnet.git
-$ cd azure-activedirectory-library-for-dotnet
+$ rename azure-activedirectory-library-for-dotnet adal.net
```

Comment:
you can condense this into a single command. "git clone git@github.com:username/azure-activedirectory-library-for-dotnet.git adal.net". this will check out the code in adal.net directory with no need to rename